src/views/TeamPage/components/Subsystems/Mechanical.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {
+    Card,
+    Container,
+    Row,
+    Col
+} from "react-bootstrap";
+import mechanical from "assets/img/mechanicaltransparent.jpg"
```

Comment:
A better way to do this is to change the opacity of the image. It gives more control and is easier.

Reply:
changed.